### DISCUSSION TITLE
SSystemExtensionErrorDomain when running local mode on mac

### DISCUSSION THREAD

**Discussion by canda** [mitmproxy/mitmproxy#6788] [Q&A] [answered]
I'm new to mitmproxy, so I might be missing something
I installed mitmproxy via brew
I run
`mitmproxy --mode local`
and got
```
Errors logged during startup:
[macos-redirector] Swift/ErrorType.swift:200: Fatal error: Error raised at top level: Error Domain=OSSystemExtensionErrorDomain Code=10 "(null)"
macos-redirector exited with status None
```
with no further information about it
any help would be appreciated

**Comment by mhils** [comment]
Do you have an MDM profile applied on your system?

**Reply by canda** [reply]
I do, via hexnode
what do I need to ask from my hexnode admins?

**Comment by mhils** [comment, marked as answer]
mitmproxy needs to be able to install a network system extension to work in
redirect local mode.

**Comment by canda** [comment]
Thank you for the help.
In my personal case, I won't be able to get permission to install this extension.
But this sounds like a very promising tool.
Great work guys, thank you
